gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/JobStatistics.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 Google Inc. All Rights Reserved.
+ * Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Why did you change these files? With my previous comment I meant that `ITGcsNio` should have 2016 copyright. It seems to me that all other files touched by this PR were created in 2015, so 2015 should be fine.

Sorry for the misunderstanding.